.circleci/config.yml

```diff
@@ -3,28 +3,21 @@
 version: 2
 jobs:
   test-1.13:
-    working_directory: /go/src/go.universe.tf/metallb
```

Comment:
Just curious: if we don't specify the working directory, how does circle CI know from which working directory to run the `go test` command specified below? I guess the default now is to run it from where your code is also copied by circle CI?

Reply:
I removed it because when changing to the "next gen" Go image from CircleCI the job was broken *with* this line and worked *without* it.

Reply:
Cool!